domain/src/main/java/edu/nextstep/camp/calculator/Calculator.kt

```diff
@@ -1,11 +1,30 @@
 package edu.nextstep.camp.calculator
 
+private typealias Chunk = List<String>
+private typealias ParsedChunk = Pair<IntArithmetics, Int>
+
 object Calculator {
+    private const val DELIMITER = " "
+    private const val CHUNK_SIZE = 2
+
     fun calculate(input: String): Int {
-        val list = input.split(" ")
-        return list.subList(1, list.size)
-            .chunked(2)
-            .map { Pair(IntArithmetics.from(it[0]), it[1].toInt()) }
-            .fold(list[0].toInt()) { acc, curr -> curr.first.apply(acc, curr.second) }
+        val (identity, words) = input.split(DELIMITER)
+        return words.chunked(CHUNK_SIZE)
+            .map { parse(it) }
```

Comment:
```suggestion
            .map(::parse)
```
전달되는 인자의 타입 및 개수와 호출하려는 함수의 인자의 타입 및 개수와 일치하는 경우에는 메서드 참조를 활용할 수 있어요.

Reply:
오 이런 문법이 가능한지는 몰랐는데 알려주셔서 감사합니다.